acceptance/testdata/secret/secret-repo-env.txtar

```diff
@@ -0,0 +1,28 @@
+# Force GitHub CLI to treat testscript as TTY
+env GH_FORCE_TTY=80
+
+# Create a repository where the secret will be registered
+exec gh repo create $ORG/$SCRIPT_NAME-$RANDOM_STRING --add-readme --private
+
+# Defer repo cleanup
+defer gh repo delete --yes $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Clone the repo
+exec gh repo clone $ORG/$SCRIPT_NAME-$RANDOM_STRING
+cd $SCRIPT_NAME-$RANDOM_STRING
+
+# Create a repository environment, will fail if organization does not have environment support
```

Comment:
**Question**

Do you think this test should run at all if the org doesn't have environment support? We could `cond` the whole test script so that it skips if there isn't support.

Reply:
That is a really good question!  We could skip this test, but I somewhat would prefer we have an environment where we don't have to worry about that and ensuring that core commands work as expected.

Reply:
Yeh so Kynan suggested we just make this work for our org, and that seems like an obvious first step. I'm not sure how that works on ghe.com, need to investigate. Beyond that I can imagine that if anyone else were to run these tests we might want to add `cond` support.

This is not something I would prioritise until other people running them was an outcome we wanted to achieve, just looking for your general opinions.

Reply:
Unfortunately I think the only way this could happen is attempting to pull plan information from the API about the organization or user account being targeted from

- [Get an organization](https://docs.github.com/en/rest/orgs/orgs?apiVersion=2022-11-28#get-an-organization)
- [Get the authenticated user](https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#get-the-authenticated-user)